pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: value_layout_builder
 description: A LayoutBuilder with an extra value. It's useful when you want to build a widget with a value computed during layout.
-version: 0.2.0
+version: 0.3.0
 homepage: https://github.com/letsar/value_layout_builder
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
-  flutter: ">=1.20.0 <2.0.0"
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=1.24.0-10.2.pre"
```

Comment:
Might make sense to change `<2.0.0` to `<3.0.0` instead of removing it if they do breaking changes in the version 3.0 🤔

Reply:
Yes, I would also recommend that

Reply:
true